### DISCUSSION TITLE
Can neorg go from file to file like vimwiki?

### DISCUSSION THREAD

**Discussion by qvieth** [nvim-neorg/neorg#452] [Q&A]
in [vimwiki](https://github.com/vimwiki/vimwiki) I can press enter on a word and it will create a link and I can move in or out of the file really quick. Does Neorg has this feature? It seems like there is nothing like this in the docs.

#Description of the mentioned feature:
Here I put the cursor here

![image](https://user-images.githubusercontent.com/71810726/167603786-4639fcb7-2ab7-4712-85d3-1eb7064b62bb.png)

Then press enter it will create a link and a file point to that link

![image](https://user-images.githubusercontent.com/71810726/167603870-3eae0089-530f-448a-b2b1-4434996580e8.png)

Enter again on the link I can move into the file, while in the file I can press backspace to move back to the last file.

**Comment by max397574** [comment]
you can add links to files and to elements in files
read `:h neorg-links` for this
a really easy way to do this is to use neorg-telescope with `insert_link` and `insert_file_link`

**Comment by qvieth** [comment]
oh sorry, I didn't search carefully not that it doesn't appear in the docs. Thank you very much for the help @max397574. I'm so appreciated.

**Comment by mrossinek** [comment]
While Max's answer is correct, I feel that it does not fully cover this discussion's question. We do _not_ currently have a mappable function which can create a file using the word under the cursor (or visual selection) for the file name. However, I too think, that this would be a valuable option to have. I'll add it to my todo list :+1:
